sdk/digitaltwins/azure-digitaltwins-core/autorest.md

```diff
@@ -39,5 +39,5 @@ java:
     models-subpackage: implementation.models
     context-client-method-parameter: true
     custom-types-subpackage: models
-    custom-types: IncomingRelationship,EventRoute,EventRoutesListOptions
```

Comment:
We used to declare these as custom types so that they would live in public namespaces, but now that we have a wrapper for each, we don't need to declare that anymore